docs/getting-started/developers/index.md

```diff
@@ -58,7 +58,7 @@ import { presetWarp } from '@warp-ds/uno';
 import { classes } from '@warp-ds/css/component-classes/classes';
 
 export default defineConfig({
-  presets: [presetWarp()],
+  presets: [presetWarp({ externalClasses: classes, skipResets: true })],
```

Comment:
For `externalClasses` to be excluded from the unocss process we'd need to set `externalizeClasses: true`, too. Otherwise `externalClasses` are not taken into account (check [this logic in drive postprocessor](https://github.com/warp-ds/drive/blob/next/src/postprocessor.js#L9)). Alternatively, we could maybe change the if-statement in drive to see if the `externalClasses` array is not empty.

Reply:
Ah, I didn't know that. Used `externalClasses: classes` on some repo's and it seems to work 🤔

Reply:
Another potential problem can occur when `components.css` that is generated and published from the newest `@warp-ds/css` is updated in Eik sooner than the package is. I mean that the aliased `v1` version was bumped but the @warp-ds/css dependency has not yet been updated in the app - `classes` array coming from that package are therefore different than the respective css available on Eik.

I don't know if this solution is possible or makes sense, but maybe we could try publishing an array of classes to Eik, so that `externalClasses` option could use such Eik url output as value. Then we know that both `components.css` and `classes` are in sync.

Reply:
Might be this @warp-ds/css PR was an attempt to handle the syncing of classes array and components.css: https://github.com/warp-ds/css/pull/56.

Reply:
externalizeClasses is true by default so inless you have set development: true - there is no need to set it.

Here is an attemp to purge classes and handle publishing https://sch-chat.slack.com/archives/C05EHJ9GELV/p1696834645308469